docs/packaging_policy.md

```diff
@@ -108,29 +108,48 @@ for each combination. There are some particular cases for this general rule:
 ## Options
 
 Recipes can list any number of options with any meaning, and defaults are up to the recipe itself. The CI cannot enforce anything
-in this direction. However, there are a couple of options that have a special meaning for the CI:
+in this direction. However, there are a couple of options that have a special meaning for the CI.
 
-* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. If it is found, it will
+### Predeinfed Options and Known Defaults
+
+ConanCenter supports many combinations, these are outline in the [supported configurations](supported_platforms_and_configurations.md) document for each platform. 
+
+* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. The **default should be `shared=False`** and will
    generate all the configurations with values `shared=True` and `shared=False`.
 
-   > Note.- The CI applies `shared=True` only to the package being built, while every other requirement will use their defaults
-   > (typically `shared=False`). It's important to keep this in mind when trying to consume shared packages from ConanCenter
+   > **Note**: The CI applies `shared=True` only to the package being built, while every other requirement will.
+   > It's important to keep this in mind when trying to consume shared packages from ConanCenter
    > as their requirements were linked inside the shared library. See [FAQs](faqs.md#how-to-consume-a-graph-of-shared-libraries) for more information.
 
-* `header_only` (with values `True` or `False`). If the CI detects this option, it will generate all the configurations for the
-   value `header_only=False` and add one more configuration with `header_only=True`. **Only one
-   package** will be generated for `header_only=True`, so it is crucial that the package is actually a _header only_ library, with header files only (no libraries or executables inside).
+* `fPIC` (with values `True` or `False`). The **default should be `fPIC=True`** and will generate all the configurations with values `fPIC=True` and `fPIC=False`.
+  This option does not make sense on all the support configurations so it should be be removed.
+
+   ```python
+   def config_options(self):
+   if self.settings.os == "Windows":
+      del self.options.fPIC
+
+   def configure(self):
+      if self.options.shared:
+         del self.options.fPIC
```

Comment:
Add the `try`/`except` idiom here, it is required for v1-v2 compatible syntax.

Reply:
Nice catch. Also, could include that is required due msvc compiler.

Reply:
@prince-chrismc please fix fmt recipe! lol

Reply:
Is it only needed for settings? The options are declared here and are not cleared AFAIK (from what I learnt reviewing I have not played with this yet)

https://github.com/conan-io/conan-center-index/blob/4b76f5816ea8812ec5c70925a05e28f51e52c030/recipes/chipmunk2d/all/conanfile.py#L36

//cc @SpaceIm

Reply:
https://github.com/conan-io/conan-center-index/pull/12034/files#diff-c89a919d270ed00a27518805fc3b5d2df6b1439c7da75901abb78b5347238095R40-R43

Ohh I just noticed this option is deleted so it might not be there

Reply:
```suggestion
            try:
                del self.options.fPIC
            except Exception:
                pass
```